README.rst

```diff
@@ -1 +1,60 @@
-initial commit
+Part of `edX code`__.
+
+__ http://code.edx.org/
+
+EdX Management Commands  |Travis|_ |Codecov|_
+===========================================
+.. |Travis| image:: https://travis-ci.org/edx/edx-management-commands.svg?branch=master
+.. _Travis: https://travis-ci.org/edx/edx-management-commands
+
+.. |Codecov| image:: http://codecov.io/github/edx/edx-management-commands/coverage.svg?branch=master
+.. _Codecov: http://codecov.io/github/edx/edx-management-commands?branch=master
+
+This repository is intended for Django 1.9 applications and utilities for reuse within
+multiple edx-platform projects.
+
+Overview
+--------
+
+This package provides a Django application containing management commands that are useful in automated
+deployment scenarios.
+
+It currently offers two management commands, ``manage_user`` and ``manage_group``, which are designed to
+facilitate idempotent read/write operations on Django users, groups, and permissions.  Consult the source
+for more specifics.
+
+Documentation
+-------------
+
+The docs for EdX Management Commands are on Read the Docs:  https://edx-management-commands.readthedocs.org.
```

Comment:
This is fine. I made the original comment before your clarification of the repo name?